### PR TITLE
Scorecard - fixes for compatiblity with new config validator

### PR DESCRIPTION
**Pull request by katze120**
- remove duplicate violations
- fix constraint name in txt output

fixes some issues in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/issues/584

### CODE REVIEW

**Comment by morgante** [comment]
@briantkennedy Wasn't this meant to be fixed in https://github.com/forseti-security/config-validator/pull/115?

**Comment by briantkennedy** [comment]
That's not 100% of it.  I opened https://github.com/forseti-security/config-validator/issues/116 to track the duplicate violation issue.  It's quite vexing because I can't repro in rego unit tests and I'm only able to repro for the full end to end FCV setup when constraint framework is involved.

File: cli/scorecard/score.go [hunk 1]
```diff
@@ -269,3 +274,17 @@ func (inventory *InventoryConfig) Score(config *ScoringConfig, outputPath string
...
+		b, _ := json.Marshal(v)
+		hash := md5.Sum(b)
+        uniqueViolationMap[string(hash[:])] = v
+    }
+    uniqueViolation := make([]*validator.Violation, 0, len(uniqueViolationMap))
+    for _, v := range uniqueViolationMap {
+        uniqueViolation = append(uniqueViolation, v)
+    }
+    return uniqueViolation
+}
```
Comment (morgante) [inline cli/scorecard/score.go]: nit. General practice is to have newline at end of files, might want to check your editor settings.

File: cli/scorecard/score.go [hunk 2]
```diff
@@ -269,3 +274,17 @@ func (inventory *InventoryConfig) Score(config *ScoringConfig, outputPath string
 
 	return nil
 }
+
+func uniqueViolation(violations []*validator.Violation) []*validator.Violation {
```
Comment (morgante) [inline cli/scorecard/score.go]:
```suggestion
func uniqueViolations(violations []*validator.Violation) []*validator.Violation {
```

File: cli/scorecard/score.go (hunk 1 above)
Comment (katze120) [reply cli/scorecard/score.go]: done!

**Pull request merged**